rpcwebsocket.go

```diff
@@ -1276,30 +1340,23 @@ func (m *wsNotificationManager) AddClient(wsc *wsClient) {
 func (m *wsNotificationManager) RemoveClient(wsc *wsClient) {
 	select {
 	case m.queueNotification <- (*notificationUnregisterClient)(wsc):
-	case <-m.quit:
+	case <-m.ctx.Done():
 	}
 }
 
-// Start starts the goroutines required for the manager to queue and process
+// run starts the goroutines required for the manager to queue and process
```

Comment:
Should improve to comment to reflect the fact that it blocks until the context is canceled (same as other servers)